web/admin/src/pages/document/component/ImportDoc/OfflineFile.tsx

```diff
@@ -127,7 +127,7 @@ const OfflineFileImport = ({
       setStep('pull-done');
       setTimeout(async () => {
         for (const { url, title } of urls) {
-          const res = await postApiV1CrawlerScrape({ url, kb_id });
+          const res = await postApiV1CrawlerScrape({ url });
```

Comment:
在OfflineFile.tsx文件中，API调用缺少必要的参数注释。

```suggestion
// 建议添加参数注释，说明kb_id参数的作用
const res = await postApiV1CrawlerScrape({ 
  url,
  // kb_id参数用于指定知识库ID
  // kb_id 
});